src/jquery.mask.js

```diff
@@ -467,6 +472,10 @@
         return this.data('mask').getCleanVal();
     };
 
+    $.fn.maskedVal = function(val) {
```

Comment:
Instead of creating a new public method wouldn't be better just returning `p.getMasked()` on the .mask public function?